standard_schema/prerelease/HED8.3.0.mediawiki

```diff
@@ -673,15 +673,17 @@ Each term in this vocabulary has a human-readable description and may include ad
 ****** Area <nowiki>[The extent of a 2-dimensional surface enclosed within a boundary.]</nowiki>
 ******* <nowiki># {takesValue, valueClass=numericClass, unitClass=areaUnits}</nowiki> 
 ****** Depth <nowiki>[The distance from the surface of something especially from the perspective of looking from the front.]</nowiki> 
-******* <nowiki># {takesValue, valueClass=numericClass, unitClass=physicalLengthUnits}</nowiki> 
-****** Length <nowiki>[The linear extent in space from one end of something to the other end, or the extent of something from beginning to end.]</nowiki> 
-******* <nowiki># {takesValue, valueClass=numericClass, unitClass=physicalLengthUnits}</nowiki> 
-****** Width <nowiki>[The extent or measurement of something from side to side.]</nowiki>
 ******* <nowiki># {takesValue, valueClass=numericClass, unitClass=physicalLengthUnits}</nowiki>
 ****** Height <nowiki>[The vertical measurement or distance from the base to the top of an object.]</nowiki>
+******* <nowiki># {takesValue, valueClass=numericClass, unitClass=physicalLengthUnits}</nowiki>
+****** Length <nowiki>[The linear extent in space from one end of something to the other end, or the extent of something from beginning to end.]</nowiki> 
 ******* <nowiki># {takesValue, valueClass=numericClass, unitClass=physicalLengthUnits}</nowiki> 
+****** Radius <nowiki>[The distance of the line from the center of a circle or a sphere to its perimeter or outer surface, respectively.]</nowiki>

```

Comment:
Are `Perimeter` and/or `Circumference` appropriate as well? `Perimeter` might work well for an irregular or bordered shape.

Reply:
(NB: I wanted to attach this to the `Size` line, but it was out of the diff's range.)

Reply:
Added Perimeter